src/main/java/lv/acodemy/homework/FirstHomeWork.java

```diff
@@ -0,0 +1,111 @@
+package lv.acodemy.homework;
+
+public class FirstHomeWork {
+    public static void main(String[] args) {
+
+        // Create 2 variable for each data type (meaningful)
+
+        // string data type
+        String name = "Inna";
+        String lastName = "Burko";
+        String fullName = name + " " + lastName;
+        System.out.println("Hi, I am" + " " + fullName + "!");
+
+        String workPosition = "lending project manager";
+        String placeOfWork = "Rietumu Banka";
+        System.out.println("I am" + " " + workPosition + " " + "in" + " " + placeOfWork);
+
+        // byte data type
+        byte myCurrentAge = 26;
+        System.out.println("I am " + myCurrentAge + " years old!");
+
+        byte loanTerm = 10;
+        System.out.println("Loan term is up to " + loanTerm + " years.");
+
+        byte numClients = 15;
+
+        // short data type
+
+        short termInMonths = 1200;
+        System.out.println("10 years is equal to " + termInMonths + " months.");
+
+        short yearWhenStartedWorking = 2017;
+        System.out.println("In " + yearWhenStartedWorking);
+
+        // char data type
+        char firstLetterOfName = 'I';
+        System.out.println(firstLetterOfName);
+
+        char firstLetterOfSurname = 66; // 'B'
+        System.out.println(firstLetterOfSurname);
+
+        // int data type
+        int loanOutstanding = 2225000;
+        System.out.println(loanOutstanding + " EUR");
+
+        int loanAmount = 500000;
+        System.out.println("Minimum loan amount from " + loanAmount + " euro");
+
+        //long data type
+        long numberOfMarksInCollection = 2147483645;
+        System.out.println(numberOfMarksInCollection);
+
+        long spedOfLight = 299792458;
+        System.out.println(spedOfLight + " metres per second");
+
+
+        //float data type
+        float interestRate = 5.5f;
+        System.out.println(interestRate + "%");
+
+        float euribor3M = 2.9f;
+        System.out.println("3M euribor on 13/03/2023 is " + euribor3M + "%.");
+
+
+        //double data type
+        double netSalarary = 855.73d;
+        System.out.println(netSalarary);
+
+        double heigh = 176.5d;
+        System.out.println(heigh);
+
+        //boolean data type
+
+        boolean itIsFirstHomeWork = true;
+        boolean itIsSecondHomeWork = false;
+        System.out.println(itIsSecondHomeWork + ", It is my first homework!");
+        System.out.println(itIsFirstHomeWork);
+
+        boolean myNameInna = true;
```

Comment:
Better to use prefix **is** here. Example: isMyNameInna and etc.

Reply:
ok!